deploy/k8s/aws/base_api-deploy.yml

```diff
@@ -13,6 +13,7 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   annotations:
+    alb.ingress.kubernetes.io/load-balancer-attributes: idle_timeout.timeout_seconds=120 # resolved 502 error while deployment "https://www.tessian.com/blog/how-to-fix-http-502-errors/"
```

Comment:
Please provide evidence this change works, and ensure you also make this change to stacks-dotnet

Reply:
Added